CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+# x.x.x (yyyy-mm-dd)
```

Comment:
I realize you're working to release an RC.  Not sure if we should include this file yet or not.  Feels a little weird having it in its current state.

Reply:
Normally this file is included in the template.  Since things got a little out of whack when trying to get this repo set up, I am including the file in this PR as part of the "initial commit".